studio/components/interfaces/Billing/PlanSelection/Plans/PlanCTAButton.tsx

```diff
@@ -20,13 +20,23 @@ const PlanCTAButton = ({ plan, currentPlan, onSelectPlan }: PlanCTAButtonProps)
   const isTeamTier = currentPlan?.supabase_prod_id === PRICING_TIER_PRODUCT_IDS.TEAM
 
   const getButtonType = (plan: any, currentPlan: any) => {
+    if (
+      currentPlan.supabase_prod_id !== PRICING_TIER_PRODUCT_IDS.ENTERPRISE &&
+      plan.name === 'Enterprise'
```

Comment:
same as above re. hardcoded string

Reply:
Component will be deleted in the next 2 weeks (everything under Billing/* which is not in V2)